### DISCUSSION TITLE
What am I doing wrong with Observable's DuckDB Client?

### DISCUSSION THREAD

**Discussion by thegilpatrick** [quarto-dev/quarto-cli#5914] [Q&A]
### Description

I have a local CSV file and I am trying to use the DuckDB client within Observable but I get an "OJS Runtime Error: memory access out of bounds". Not sure what I am doing wrong here?

````qmd
---
title: reproducible quarto doc
format: html
---

```{ojs}
db = DuckDBClient.of({
  presidents: FileAttachment("election_data.csv")
})

data = db.sql`SELECT * FROM presidents`

data
```
````
Quarto: 1.3.159
VSCode 1.79.1
Windows 10

**Comment by jimjam-slam** [comment]
Hey @thegilpatrick, do you have a copy of `election_data.csv` that others can test? I'm able to render equivalent code with [`ipsos-trust-professions-bycountry.csv`](https://github.com/360-info/report-policingthepolice/blob/main/data/ipsos-trust-professions-bycountry.csv), which was a CSV I had lying around (assuming that CSV file is in the same directory as the `.qmd` file!):

````
---
title: reproducible quarto doc
format: html
---

```{ojs}
db = DuckDBClient.of({
  trust: FileAttachment("ipsos-trust-professions-bycountry.csv")
})

data = db.sql`SELECT * FROM trust`

data
```
````

**Reply by thegilpatrick** [reply]
Sorry about that! Yes it is in the same directory as the `.qmd` file. Here is the [election_data.csv](https://raw.githubusercontent.com/thegilpatrick/election_data/main/election_data.csv) file.

**Comment by cscheid** [comment]
Huh. That's a weird one. `memory access out of bounds` sounds like an exception coming out of webassembly, which makes me suspect this is not an issue with Quarto.
